docs/core/diagnostics/dotnet-sos.md

```diff
@@ -56,9 +56,22 @@ Installs the [SOS extension](../../framework/tools/sos-dll-sos-debugging-extensi
 ### Synopsis
 
 ```console
-dotnet-sos install
+dotnet-sos install [--architecture <arch>]
 ```
 
+### Options
+
+- **`--architecture <arch>`**
+
+  Specifies the processor architecture of the SOS binaries to install. By default, `dotnet-sos` installs the architecture of the host machine. Use this option when you are trying to install SOS for a different architecture from your host architecture - for example, if you are running Arm32 binaries from an Arm64 host, you will need to install SOS with `dotnet-sos install --architecture Arm`.
```

Comment:
nit: "different architecture form your host architecture" -> "different from the dotnet host architecture"